l10n_ro_partner_create_by_vat/views/partner_view.xml

```diff
@@ -2,24 +2,24 @@
 <openerp>
     <data>
         <!-- Partner -->
-        <record model="ir.ui.view" id="partners_form_address">
+        <record id="partners_form_address" model="ir.ui.view">
             <field name="name">partners.form.address</field>
             <field name="model">res.partner</field>
             <field name="inherit_id" ref="base.view_partner_form" />
             <field eval="100" name="priority" />
             <field name="arch" type="xml">
-                <field name="name" position="after">
+                <field name="ref" position="after">
```

Comment:
Cred ca ar fi ok sa fie in amandoua view-urile dupa website, sau dupa alt camp ca sa fie la fel...